enterprise/dockerfiles/ci_runner_image/Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM gcr.io/cloud-marketplace/google/debian10@sha256:c571e553cdaa91b1f16c190a049ccef828234ac47a0e8ef40c84240e62108591
+FROM gcr.io/cloud-marketplace/google/rbe-ubuntu16-04@sha256:5464e3e83dc656fc6e4eae6a01f5c2645f1f7e95854b3802b85e86484132d90e
```

Comment:
16.04 is already EOL. Can we base this on a bazel-ified 18.04 or 20.0x?

Reply:
Done